src/views/deployments/DeploymentPages.tsx

```diff
@@ -138,3 +144,43 @@ export const CloneDeploymentPage: React.FC = () => {
         </FetchingEntity>
     );
 };
+
+export const CreateFromPackagingPage: React.FC = () => {
```

Comment:
I thought that the component would be the same as for new Deployment, but with prefilled data provided. Could you explain why do we need separate components for "Deployment from Packaging" or "Cloning"?

Reply:
This is not a component in terms of reusability. This is a specific view. The components that imply reusability is <EditablePage> and <FetchingEntity>.
The fact that there is some similar code/behavior in different views is not an unambiguous reason to combine them into some generic, imo